packages/components/buttons/primary-button/src/primary-button.tsx

```diff
@@ -129,7 +152,10 @@ const PrimaryButton = <TStringOrComponent extends ElementType = 'button'>(
             {props.iconLeft &&
               cloneElement(props.iconLeft, {
                 color: props.isDisabled ? 'neutral60' : 'surface',
-                size: props.size === 'big' ? 'big' : 'medium',
+                size:
+                  props.size === 'big' || props.size === '20'
+                    ? 'big'
+                    : 'medium',
```

Comment:
Considering I didn't participate in the discussions about the task, I hope I got everything right. 
Are we only deprecating T-shirt sizes for icon buttons, or for icons as well?
If we do that for icons, I wonder, shouldn't the ternary return the new sizes? And then we could also update https://github.com/commercetools/ui-kit/blob/6ca49442df321fe6e1142c0f36fed073177b019a/packages/components/icons/src/icons.visualroute.jsx#L47 and https://github.com/commercetools/ui-kit/blob/6ca49442df321fe6e1142c0f36fed073177b019a/packages/components/icons/src/icon.story.js#L81.
Otherwise please disregard

Reply:
Indeed, it is a little ambiguous. The ticket mentions introducing the new sizes to the buttons, figma makes it appear as if Icons should get them as well. 

@FilPob Should Icons get the numeric sizing scale as well?

Reply:
Thanks! Just for the record, regardless of the ticket's title, this PR already [covers T-shirt size deprecation in all icons](https://github.com/commercetools/ui-kit/blob/1f5223bc86a985d4fa531f8356f31f27ba76f736/packages/components/icons/src/templates/icon.styles.tsx#L18-L22), so I just wanted to make sure that's also expected here.

Reply:
You're right, Kacper 👍 

Then I agree with you we should use the new sizes values.

Reply:
It also makes sense, considering that LeadingIcon and CustomIcon are already using the new scale-system. However, one problem that I see is, the cleanup afterwards. Cause ideally I'd want to make sure that no icon in ui-kit is used anywhere with one of the old tokens. But since I am merely editing a template, and then generate 147 standalone react-components from it, this will be extremely time-consuming.

Reply:
I just did a quick implementation, comparable to the button solution (sizeMapping old:new & console warning-helper), resulting in 583 broken tests due to the console.warn.

Any suggestions?

Reply:
I would suggest to change in this PR only the two places Kacper pointed out and let the cleanup for a follow-up PR so we keep the scope of this one contained.

Since I'm FLD this week, I can help with the cleanup (I don't think it would take me much time).